client/src/templates/Challenges/components/help-modal.tsx

```diff
@@ -53,61 +87,198 @@ function HelpModal({
+  const [showHelpForm, setShowHelpForm] = useState(false);
+  const [description, setDescription] = useState('');
+  const [readSearchCheckbox, setReadSearchCheckbox] = useState(false);
+  const [similarQuestionsCheckbox, setSimilarQuestionsCheckbox] =
+    useState(false);
+  const DESCRIPTION_MIN = 50;
+  const DESCRIPTION_MAX = 500;
+
+  const canSubmitForm = useCallback(() => {
+    return (
+      description.length >= DESCRIPTION_MIN &&
+      readSearchCheckbox &&
+      similarQuestionsCheckbox
+    );
+  }, [description, readSearchCheckbox, similarQuestionsCheckbox]);
+
+  const resetFormValues = () => {
+    setDescription('');
+    setReadSearchCheckbox(false);
+    setSimilarQuestionsCheckbox(false);
+  };
+
+  const handleCheckboxChange = (
+    event: React.ChangeEvent<HTMLInputElement>,
+    set: React.Dispatch<React.SetStateAction<boolean>>
+  ) => {
+    set(event.target.checked);
+  };
+
   if (isOpen) {
     callGA({ event: 'pageview', pagePath: '/help-modal' });
   }
   return (
-    <Modal dialogClassName='help-modal' onHide={closeHelpModal} show={isOpen}>
+    <Modal
+      dialogClassName='help-modal'
+      onHide={() => {
+        closeHelpModal();
+        setShowHelpForm(false);
+        resetFormValues();
+      }}
+      show={isOpen}
+    >
       <Modal.Header className='help-modal-header fcc-modal' closeButton={true}>
         <Modal.Title className='text-center'>
           {t('buttons.ask-for-help')}
         </Modal.Title>
       </Modal.Header>
       <Modal.Body className='help-modal-body text-center'>
-        <h3 className='help-modal-heading'>
-          <Trans i18nKey='learn.tried-rsa'>
-            <a
-              href={RSA}
-              rel='noopener noreferrer'
-              target='_blank'
-              title={t('learn.rsa')}
+        {showHelpForm ? (
+          <form
+            onSubmit={event => {
+              event.preventDefault();
+              if (!canSubmitForm()) return;
+              setShowHelpForm(false);
+              resetFormValues();
+              createQuestion();
+              closeHelpModal();
+            }}
+          >
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'column'
+              }}
             >
-              placeholder
-            </a>
-          </Trans>
-        </h3>
-        <div className='alert alert-danger'>
-          <FontAwesomeIcon icon={faExclamationCircle} />
-          <p>
-            <Trans i18nKey='learn.rsa-forum'>
-              <a
+              <CheckboxHelpModal
+                name='read-search-ask-checkbox'
+                i18nkey={t('learn.read-search-ask-checkbox')}
+                onChange={event =>
+                  handleCheckboxChange(event, setReadSearchCheckbox)
+                }
+                value={readSearchCheckbox}
+                href={RSA}
+                t={t}
+              />
+              <CheckboxHelpModal
```

Comment:
```suggestion
              <Spacer size='small' />
              <CheckboxHelpModal
```

It would be nice to add a little more space between these checkboxes to increase the target size for touch devices.